hip/settings/dev.py

```diff
@@ -3,6 +3,8 @@
 from .base import *  # noqa
 
 
+STATICFILES_STORAGE = "django.contrib.staticfiles.storage.StaticFilesStorage"
```

Comment:
This needs to be inside of a conditional so that it is applied only during tests (we want to use whitenoise during dev to make sure that we see as many prod issues as possible during local development). So something like this:
```
if "pytest" in sys.modules:
    STATICFILES_STORAGE = "django.contrib.staticfiles.storage.StaticFilesStorage"
```

Reply:
done